modes/longitudinal/src/toolbarButtons.js

```diff
@@ -295,26 +296,41 @@ const toolbarButtons = [
     },
   },
   {
-    id: 'Layout',
-    type: 'ohif.layoutSelector',
-    props: {
-      rows: 3,
-      columns: 3,
-    },
-  },
-  {
-    id: 'MPR',
-    type: 'ohif.action',
+    id: 'Layouts',
+    type: 'ohif.splitButton',
     props: {
-      type: 'toggle',
-      icon: 'icon-mpr',
-      label: 'MPR',
-      commands: [
-        {
-          commandName: 'toggleMPR',
-          commandOptions: {},
-          context: 'CORNERSTONE',
+      isRadio: false,
+      groupId: 'Layouts',
+      primary: {
+        id: 'Layout',
+        type: 'action',
+        uiType: 'ohif.layoutSelector',
+        props: {
+          rows: 3,
+          columns: 3,
         },
+      },
+      secondary: {
+        icon: 'chevron-down',
+        label: '',
+        isActive: true,
+        tooltip: 'More Layouts',
+      },
+      items: [
```

Comment:
Eventually we will need a computed child list rather than a static one - something that can list various stages as well as other hanging protocols.  Should think about how to handle that, but probably it doesn't belong here right now.

Reply:
Yes, perhaps later.